components/event-bus/cmd/event-publish-service/publisher/publisher.go

```diff
@@ -14,8 +14,10 @@ import (
 const (
 	// FAILED status label
 	FAILED = "failed"
-	// IGNORED status label
-	IGNORED = "ignored"
+	// IGNORED_NO_CHANNEL status label
+	IGNORED_NO_CHANNEL = "ignored_no_channel"
+	// IGNORED_NOT_READY_CHANNEL status label
+	IGNORED_NOT_READY_CHANNEL = "ignored_not_ready_channel"
```

Comment:
can we make it sound more like natural language and less like :yoda:
`IGNORED_CHANNEL_NOT_READY`

Reply:
Done